src/components/ConfirmedRoute.js

```diff
@@ -43,7 +44,7 @@ const getWaypointMarkers = (waypoints) => {
     const lastWaypointIndex = numberOfWaypoints - 1;
     return _.filter(
         _.map(waypoints, (waypoint, key) => {
-            if (!waypoint || waypoint.lng === undefined || waypoint.lat === undefined) {
+            if (!waypoint || lodashIsNil(lodashGet(waypoint, 'lng')) || lodashIsNil(lodashGet(waypoint, 'lat'))) {
```

Comment:
NAB: can this be `if (!waypoint || !waypoint.lng || !waypoint.lat)`?

Reply:
when we have lat/lng as 0 value  also ignore here

Reply:
Right, @neil-marcellini this approach will treat `lat/lng = 0` as falsy and cause the early return.
Even though it's highly unlikely that someone will choose a point with a latitude or longitude equal exactly to 0, the probability is never 0.

How about this?
```js
if (!waypoint || lodashIsNil(waypoint.lng) || lodashIsNil(waypoint.lat))
```

Reply:
incase waypoint.lng is `undefined`  means lodashIsNil returns false only.

Reply:
No, `lodashIsNil` returns `false` if the value is nullish, meaning `null` or `undefined`.

Reply:
@paultsimura lodashIsNil(null)  =  true , lodashIsNil(undefined)  =  false.  
>lodashIsNil returns false if the value is nullish, 

that is what i am mention in my comment. ` lodashIsNil(waypoint.lng)` if value is `undefined`  means early return will be ignored here.

`lodashIsNil`  will be return true only value have null.

Reply:
@pradeepmdk please take a look: https://lodash.com/docs/#isNil

Reply:
@pradeepmdk please take a look: https://lodash.com/docs/#isNil

Reply:
Ah right I keep forgetting they can be 0. Sounds good.